pkg/download/os_arch.go

```diff
@@ -26,7 +26,11 @@ func isThisRuntimeRelease(r string) bool {
 
 func isMatchingReleaseName(r, o, a string) bool {
 	r = strings.ToLower(r)
-	archOK := false
+	// TODO: fix this function. Not all releases provide an arch.
```

Comment:
I just nop'd this out for dev reasons, but we should probably implement a better fix.